steward/src/cellars/adaptors/morpho.rs

```diff
@@ -308,6 +312,203 @@ pub(crate) fn morpho_aave_v3_debt_token_adaptor_v1_calls(
+                let market = steward_abi::morpho_blue_collateral_adaptor_v1::MarketParams {
+                    loan_token: sp_call_parse_address(market.loan_token)?,
+                    collateral_token: sp_call_parse_address(market.collateral_token)?,
+                    oracle: sp_call_parse_address(market.oracle)?,
+                    irm: sp_call_parse_address(market.irm)?,
+                    lltv: string_to_u256(market.lltv)?,
+                };
+                let call = steward_abi::morpho_blue_collateral_adaptor_v1::AddCollateralCall {
+                    market,
+                    collateral_to_deposit: string_to_u256(p.collateral_to_deposit)?,
+                };
+                calls.push(
+                    MorphoBlueCollateralAdaptorV1Calls::AddCollateral(call)
+                        .encode()
+                        .into(),
+                )
+            }
+            morpho_blue_collateral_adaptor_v1::Function::RemoveCollateral(p) => {
+                let market = p.market.unwrap();
+                let market = steward_abi::morpho_blue_collateral_adaptor_v1::MarketParams {
+                    loan_token: sp_call_parse_address(market.loan_token)?,
+                    collateral_token: sp_call_parse_address(market.collateral_token)?,
+                    oracle: sp_call_parse_address(market.oracle)?,
+                    irm: sp_call_parse_address(market.irm)?,
+                    lltv: string_to_u256(market.lltv)?,
+                };
+                let call = steward_abi::morpho_blue_collateral_adaptor_v1::RemoveCollateralCall {
+                    market,
+                    collateral_amount: string_to_u256(p.collateral_amount)?,
+                };
+                calls.push(
+                    MorphoBlueCollateralAdaptorV1Calls::RemoveCollateral(call)
+                        .encode()
+                        .into(),
+                )
+            }
+        }
+    }
+
+    Ok(calls)
+}
+
+pub(crate) fn morpho_blue_debt_adaptor_v1_calls(
+    params: steward_proto::steward::MorphoBlueDebtAdaptorV1Calls,
+) -> Result<Vec<Bytes>, Error> {
+    let mut calls = Vec::new();
+    for c in params.calls {
+        let function = c
+            .function
+            .ok_or_else(|| sp_call_error("function cannot be empty".to_string()))?;
+
+        match function {
+            morpho_blue_debt_adaptor_v1::Function::RevokeApproval(p) => {
+                let call = steward_abi::morpho_blue_debt_adaptor_v1::RevokeApprovalCall {
+                    asset: sp_call_parse_address(p.asset)?,
+                    spender: sp_call_parse_address(p.spender)?,
+                };
+                calls.push(
+                    MorphoBlueDebtAdaptorV1Calls::RevokeApproval(call)
+                        .encode()
+                        .into(),
+                )
+            }
+            morpho_blue_debt_adaptor_v1::Function::BorrowFromMorphoBlue(p) => {
+                let market = p.market.unwrap();
```

Comment:
We probably shouldn't be calling `unwrap` and introducing the possibility of a panic. We can have the market params converting function sanity check that the field is present and return an error otherwise, like the other parse functions do.